spec/datetimeformat.html

```diff
@@ -265,6 +276,12 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
         1. Perform ! CreateDataPropertyOrThrow(_nf2Options_, `"minimumIntegerDigits"`, 2).
         1. Perform ! CreateDataPropertyOrThrow(_nf2Options_, `"useGrouping"`, *false*).
         1. Let _nf2_ be ? Construct(%NumberFormat%, &laquo; _locale_, _nf2Options_ &raquo;).
+        1. Let _fractionalSecondDigits_ be _dateTimeFormat_.[[FractionalSecondDigits]]
+          1. If _fractionalSecondDigits_ is greater than 0, then
```

Comment:
Should test that `fractionalSecondDigits` is not `undefined`. And elsewhere we should document that `fractionalSecondDigits` can't be zero for resolved patterns.